receiver/hostmetricsreceiver/README.md

```diff
@@ -134,6 +136,34 @@ The following settings are optional:
 - `mute_process_exe_error` (default: false): mute the error encountered when trying to read the executable path of a process the collector does not have permission to read (Linux only). This flag is ignored when `mute_process_all_errors` is set to true as all errors are muted.
 - `mute_process_user_error` (default: false): mute the error encountered when trying to read a uid which doesn't exist on the system, eg. is owned by a user that only exists in a container. This flag is ignored when `mute_process_all_errors` is set to true as all errors are muted.
 
+### GroupProcess
+The `groupprocessscraper` collects metrics for groups of processes based on user given regex configurations. This allows for more granular monitoring of specific sets of processes.
+
+```yaml
+receivers:
+  hostmetrics:
+    collection_interval: 10s
+    scrapers:
+      groupprocess:
+        process_configs:
+          - group_name: "kube_process"
+            comm:
+              names:
+                - "kube-proxy"
+              match_type: "strict"
+            exe:
+              names:
+                - "/usr/local/bin/kube-proxy"
+              match_type: "strict"
+            cmdline:
+              names:
+                - "--config=/var/lib/kube-proxy/config.conf"
+              match_type: "regexp"
+```
+In this example, the groupprocessscraper is configured to scrape metrics for processes that match the specified comm, exe, and cmdline criteria. The group_name is used to identify the group of processes being monitored.
+For `comm` and `exe`, the list of strings is an `OR`, meaning any process matching any of the strings will be added to the process group.
+For `cmdline`, the list of regexes is an `AND`, meaning they all must match.
+
```

Comment:
Also, add a line.
If more than one out of(comm, exe and cmdline) is given all of them has to match.

if a process matches to one group, the same process will not be part of other groups.